### ISSUE TITLE
Filter explain

### ISSUE THREAD

**Issue by Dreadaxe** [ianmaddox/feedsifter#1]
Can I filter by description, title, or category word?

**Comment by ianmaddox** [comment]
Seems reasonable. How about something like:

> globalterm
> Title:/titleterm/
> Title:/anothertitleterm/

It would be mostly backwards compatible and extensible.

**Comment by Dreadaxe** [comment]
Excellent. 

Category term is important too.

**Comment by dvolatility** [comment]
Hi Ian, is this title filter still being considered because I need it badly! I tried using your sifter with just a snippet feed and posts are still being triggered because feed categories are being jammed with keywords. Or, is there a way to make it so categories don't get triggered in your sifter? Thanks.

**Actually I think I found a way to not use the feedburner feed with all the categories.

**Comment by ianmaddox** [comment]
@dvolatility Yep, it's a great feature, but I have a lot of other things ahead of it in my project queue. If someone wants to send a PR, I'll happily take it in.

**Comment by Dreadaxe** [comment]
Example of this rss feed
https://www.tabletoptournaments.net/fr/t3_tournaments.xml

I want filter  category `<tournaments:type>` or `<category>` how I can do?

**Comment by mepcovenkat97** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
marcinek89xl, @CM270, @Mr-sleepy-J, @scnale, @zihengCat, @porad8, @Bekitzer, @Faizan711, @traviskaufman, @haneyume, @YourDailyBlockchain, @cryptickoan, @sibelius, @hlhdaiaaii, @PataMali, @mfer8023, @Mereson, @LucasReSilva, @kentng201, @nekofar
